tests/baselines/reference/thisTypeInClasses.errors.txt

```diff
@@ -1,57 +0,0 @@
-tests/cases/conformance/types/thisType/thisTypeInClasses.ts(4,20): error TS2526: A 'this' type is available only in a non-static member of a class or interface.
-
-
-==== tests/cases/conformance/types/thisType/thisTypeInClasses.ts (1 errors) ====
-    class C1 {
-        x: this;
-        f(x: this): this { return undefined; }
-        constructor(x: this) { }
-                       ~~~~
-!!! error TS2526: A 'this' type is available only in a non-static member of a class or interface.

```

Comment:
This particular test is supposed to have no errors. We generate .symbols and .types files only when there are no errors and part of this test is to check that those files remain constant.